pkg/backup/backup/backup_manager.go

```diff
@@ -366,17 +366,48 @@ func (bm *backupManager) makeBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, s
 		})
 	}
 
+	brVolume := corev1.VolumeMount{
+		Name:      "br-bin",
+		ReadOnly:  false,
+		MountPath: util.BRBinPath,
+	}
+	volumeMounts = append(volumeMounts, brVolume)
+
+	volumes = append(volumes, corev1.Volume{
+		Name: "br-bin",
+		VolumeSource: corev1.VolumeSource{
+			EmptyDir: &corev1.EmptyDirVolumeSource{},
+		},
+	})
+
 	serviceAccount := constants.DefaultServiceAccountName
 	if backup.Spec.ServiceAccount != "" {
 		serviceAccount = backup.Spec.ServiceAccount
 	}
+
+	brImage := "pingcap/br:" + tikvVersion
```

Comment:
pingcap/br -> pingcap/tidb-lightning, br will not publish with the tidb images, please test it.

Reply:
@shuijing198799 according to @DanielZhangQD (https://github.com/pingcap/tidb-operator/issues/3441#issuecomment-720931439) the BR image has been released and I checked the [docker hub](https://hub.docker.com/r/pingcap/br/tags), it has all the versions since v3.1. I tested with this too.

Reply:
@namco1992 Yes, please use `pingcap/br`.

Reply:
OK it seems more reasonable